src/main/java/codurance/academyfinalboy/backend/model/team/UserRef.java

```diff
@@ -0,0 +1,17 @@
+package codurance.academyfinalboy.backend.model.team;
+
+import lombok.Data;
+import org.springframework.data.jdbc.core.mapping.AggregateReference;
+import org.springframework.data.relational.core.mapping.Column;
+import org.springframework.data.relational.core.mapping.Table;
+
+@Data
+@Table("user_team")
```

Comment:
Would this be some kind of lookup table? You can resolve this comment after answering is just a doubt.